cpp/dolfin/CMakeLists.txt

```diff
@@ -88,6 +88,8 @@ target_include_directories(dolfin SYSTEM PUBLIC ${UFC_INCLUDE_DIRS})
 
 # Eigen3
 target_include_directories(dolfin SYSTEM PUBLIC ${EIGEN3_INCLUDE_DIR})
+# Force Eigen to use 32-byte alignment for compatibility with AVX code
+target_compile_definitions(dolfin PUBLIC EIGEN_MAX_ALIGN_BYTES=32)
```

Comment:
This should be a CMake option (so the user can set it), with a default value.

Reply:
Fixed. Not sure about naming though. Using `EIGEN_MAX_ALIGN_BYTES` suggest IMO that it only affects Eigen itself, whereas it actually also affects coupling with user code. Therefore, called it `DOLFIN_ALIGNMENT_BYTES`